### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by eeeeeffgghhyjuui** [interstellarnetwork/interstellar#7]
### What is this request?

New game

### Please describe the new feature here

Minecraft unblocked, name of site is Eaglercraft.org, there is an option on the site to download as an HTML file

**Comment by Animationking11** [comment]
They could add it via https://github.com/PrecisionClient/PrecisionClient-Static all they would have to do is copy the files and put it in a folder

**Comment by imbubbo** [comment]
> They could add it via https://github.com/PrecisionClient/PrecisionClient-Static all they would have to do is copy the files and put it in a folder

Already Done
